tests/common.sh

```diff
@@ -46,6 +46,10 @@ function populate_test_list {
 }
 
 function wait_clean {
+  if [[ `kubectl get benchmarks.ripsaw.cloudbulldozer.io --all-namespaces` ]]
+  then
+    kubectl delete benchmarks -n my-ripsaw --all --ignore-not-found
```

Comment:
This is not necessary, deleting the namespace also deletes the benchmarks from that namespace.

Reply:
@rsevilla87 i ran into an issue yesterday where the namespaces were gone but there were still benchmarks:

```
 # oc get benchmarks.ripsaw.cloudbulldozer.io -n my-ripsaw-6c0ed098-b139-4420-ac7a-63d2bd2b96cf-test-fs-drift 
NAME                TYPE       STATE      METADATA STATE   CERBERUS        UUID                                   AGE
example-benchmark   fs-drift   Complete   Collecting       not connected   b03d2cc2-d999-5fbd-93a8-406f3247986e   22m
# oc get namespaces my-ripsaw-6c0ed098-b139-4420-ac7a-63d2bd2b96cf-test-fs-drift
Error from server (NotFound): namespaces "my-ripsaw-6c0ed098-b139-4420-ac7a-63d2bd2b96cf-test-fs-drift" not found
```

Reply:
lol. You have my rubber stamp in that case.